onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -619,4 +667,23 @@ def sample_inputs_tensor_int(op_info, device, dtype, requires_grad, **kwargs):
         dtypes=common_dtype.all_types_and(torch.half, torch.bfloat16),
         sample_inputs_func=sample_inputs_tensor_int,
     ),
+    opinfo_core.OpInfo(
+        # Unique ID used to connect
+        #  TorchLibOpInfo("aten.bernoulli.p", ...)
+        # and
+        #  opinfo_core.OpInfo("aten.bernoulli.p", ...)
+        "aten.bernoulli.p",
+        aten_name="bernoulli.p",
+        op=torch.ops.aten.bernoulli.p,
+        # dtypes can be a tuple of (torch.float, torch.double).
+        dtypes=common_dtype.all_types(),
+        sample_inputs_func=sample_inputs_bernoulli_p,
+    ),
+    opinfo_core.OpInfo(
+        "aten.bernoulli.default",
```

Comment:
Possible to just use the torch provided bernoulli?

Reply:
Sure. Note that this implies a fundamental problem -- ONNX script doesn't support overloads but PyTorch (e.g., torch.ops.aten.bernoulli(...) can invoke any of torch.ops.aten.bernoulli.*) does. I will leave it to you.

Reply:
Humm, so which op's exporter should use this `OpInfo`? If it contains two operators (aten.bernoulli and aten.bernoulli.p), how do I know which exporter to check if there is an error?

Reply:
TBD: I will leave this untouched since I feel the right solution is to write test data for each overload independently. Mixing test data for different overloads will complicate debugging experience.

Reply:
There’s existing OpInfo for Bernoulli se we don’t need to create our own. The benefit of using existing tests from torch is when then refine their test cases, we get the updates for free. I can create a follow up PR.

Reply:
I created https://github.com/microsoft/onnxscript/pull/850